pkg/pillar/cmd/volumemgr/handleverifier.go

```diff
@@ -233,3 +233,23 @@ func handleVerifyImageStatusDelete(ctxArg interface{}, key string,
 	updateStatus(ctx, status.ImageSha256)
 	log.Infof("handleVerifyImageStatusDelete done for %s", key)
 }
+
+//handleInitBlobVerifyImgRef during volumeMgr init, we create and publish blobStatus for all the blobs present in CAS.
+// If there is a verified image found for a blob, this method adds a ref to the respective blobStatus.
```

Comment:
If you already have a BlobStatus with IsLoaded set, why do you care about keeping a RefCount on the VerifyImageStatus? Once it is loaded into CAS we should be able to discard the VerifyImageConfig+Status to free up redudant usage of space.

Reply:
I think we should just add another `SwState` that is higher than `VERIFIED` that is `LOADED`, and we can work off of that `BlobStatus.State`

Reply:
That would make sense. Not that we need to map that to the same ZSwState in zedagent since we don't want to add a new state to the API; too painful to roll that out for minor things like this.

Reply:
No, I don't think it needs to go upstream. Upstream's level of interest should end once it has a verified blob. What happens on the device to move it around is not of interest to it.

Reply:
Changed. Now on volumeMgr init, after we receive and handle all VerifyImageStatus, ContentTreeConfig and blob in CAS, any unused blob and verifiedImge is deleted.